core/src/script/CGXP/plugins/MyPosition.js

```diff
@@ -61,6 +61,31 @@ cgxp.plugins.MyPosition = Ext.extend(gxp.plugins.Tool, {
      */
     recenteringZoom: 11,
 
+    /** api: config[drawAccuracy]
+     *  ``Boolean``
+     *  Condition for drawing the circle of accuracy, default is false. 
+     */
+    drawAccuracy: false,
+
+    /** private: property[stylePoint]
+    * ``Object`` Feature style hash to apply to the
```

Comment:
Why are "stylePoint" and "styleAccuracy" private properties? Shouldn't they be available in the API?